Tests/Unit/Sync/JobsCleanupServiceTest.php

```diff
@@ -116,12 +117,15 @@ public function testInvalidTableName($tableName)
      */
     public function getTableNameData()
     {
-        // Case #0 incorrect table name.
+        // Case #0 .
```

Comment:
Definition is necessary here and 2 below. It should describe initial condition and expected test outcome.